app/src/main/java/com/paddlesandbugs/dahdidahdit/brasspound/HardwareKeySensor.java

```diff
@@ -19,22 +19,27 @@
 package com.paddlesandbugs.dahdidahdit.brasspound;
 
 import android.view.KeyEvent;
+import android.view.MotionEvent;
 
 public class HardwareKeySensor {
 
     private final int keyCode;
     private final Keyer keyer;
     private final int key;
+    public final boolean isMouse;
 
 
-    public HardwareKeySensor(int keyCode, Keyer keyer, int key) {
+    public HardwareKeySensor(int keyCode, Keyer keyer, int key, boolean isMouse) {
         this.keyCode = keyCode;
         this.keyer = keyer;
         this.key = key;
+        this.isMouse = isMouse;
     }
 
 
     public void dispatchKeyEvent(KeyEvent e) {
+        if (isMouse) return;
```

Comment:
Please wrap blocks in if/else statements in curly braces - even if they are only one-liners.